main.py

```diff
@@ -351,5 +297,15 @@ def run(self):
 
 
 if __name__ == '__main__':
-    sdw = SiggyDeltaWarnings()
+    if (WEBHOOK_TYPE == 'discord'):
+        web_hook = WebHookAdapter.WebHookAdapter(WebHookAdapter.DiscordWebHook())
+    else:
+        web_hook = WebHookAdapter.WebHookAdapter(WebHookAdapter.SlackWebHook())
+
+    if (SOURCE_TYPE == 'eve-scout'):
+        route_source = RouteSourceAdapter.RouteSourceAdapter(RouteSourceAdapter.EveScoutSource())
```

Comment:
Same here, a nice utility to avoid these long lines with nasty imports that are hard to read.